### DISCUSSION TITLE
Version Compatibility

### DISCUSSION THREAD

**Discussion by seanphilipp** [OpenRVS-devs/OpenRVS#53] [Q&A]
Hello,

I'd like to understand how the OpenRVS ~~patches~~ versions work wrt (backward) compatibility. i.e. Does a server running 1.5 support users running anterior versions?

Thank you!

**Comment by willroberts** [comment]
> Does a server running 1.5 support users running anterior versions?

Yes! Our testing process includes using clients from the current release against new servers, and using clients from the new release against current servers in order to ensure compatibility.
